tracker2.rst

```diff
@@ -154,67 +170,53 @@ Doing all of this allows core developers and :ref:`triagers <devrole>` to more
 quickly look for subtle issues that only people with extensive experience
 working on Python's code base will notice.
 
-
 Finding an Issue You Can Help With
 ----------------------------------
 
 If you want to help triage issues, you might also want to search for issues
 in modules which you have a working knowledge.  Search for the name of a module
-in the issue tracker or use the `advanced search`_ to search for specific 
-components (e.g. "Windows" if you are a Windows developer, "Extension Modules"
-if you are familiar with C, etc.). Finally you can use the "Random issue" link
-in the sidebar to pick random issues until you find an issue that you like.  
-You may find old issues that can be closed, either because they
-are no longer valid or they have a patch that is ready to be committed, 
-but no one has had the time to do so.
-
-In the sidebar you can also find links to summaries for easy issues and
-issues with a patch.
+in the issue tracker or use the `advanced search`_ query builder to search for
+specific kinds of issues (e.g. the "Windows" label if you are a Windows
+developer, "Extension Modules" if you are familiar with C, etc.). 
 
 
 .. _devrole:
 
-Gaining the "Developer" Role on the Issue Tracker
-=================================================
+Gaining the "Triager" Role on the Issue Tracker
```

Comment:
I think we can remove this whole section completely. I believe this was specific to the role on b.p.o.

For becoming a member of the Python Triage team on GitHub, it was already covered in a different page: https://devguide.python.org/triaging/#becoming-a-member-of-the-python-triage-team